src/__test__/persistentSubscription/getPersistentSubscriptionInfo.test.ts

```diff
@@ -0,0 +1,160 @@
+import { createTestNode, delay, jsonTestEvents } from "@test-utils";
+
+import {
+  END,
+  EventStoreDBClient,
+  persistentSubscriptionSettingsFromDefaults,
+  ROUND_ROBIN,
+  START,
+} from "@eventstore/db-client";
+
+describe("getPersistentSubscriptionInfo", () => {
+  const node = createTestNode();
+  let client!: EventStoreDBClient;
+
+  beforeAll(async () => {
+    await node.up();
+
+    client = new EventStoreDBClient(
+      {
+        endpoint: node.uri,
+      },
+      { rootCertificate: node.rootCertificate },
+      { username: "admin", password: "changeit" }
+    );
+  });
+
+  afterAll(async () => {
+    await node.down();
+  });
+
+  test("should get info on a persistant subscription", async () => {
```

Comment:
**TYPO:** The same applies to other test names.
```suggestion
  test("should get info on a persistent subscription", async () => {
```

Reply:
Fixed: https://github.com/EventStore/EventStore-Client-NodeJS/pull/273/commits/339ec2a164c0cd69bc5386797e3aee3ed65f184d

:face_exhaling: